### ISSUE TITLE
Problem running the version download from the website 

### ISSUE THREAD

**Issue by marcpiulachs** [hzcx998/XBook#10]
![image](https://user-images.githubusercontent.com/2106186/77436590-f560ff80-6de3-11ea-9ad6-47b7f69adb00.png)

**Comment by hzcx998** [comment]
I think it may be the video mode problem, you can change the video mode to VMODE_800_600_16 and try again.

The code is at arch/x86/include/boot/config.inc, and I will give you a picture.

After change that, you should input `make clean`, and `make run` again.












------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"boss"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月24日(星期二) 晚上10:28
收件人:&nbsp;"hzcx998/XBook"<XBook@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[hzcx998/XBook] Problem running the version download from the website  (#10)

**Comment by marcpiulachs** [comment]
Yes but remember I'm using the downloaded version, if I make the changes to the source code and do 'make' again the c.img will not be created as it's creation is commented on the makefile

**Comment by hzcx998** [comment]
I mean, you download the source code to copy the image you clone from github source, so that you can use the image locally, because gitignore is the image file is ignored, this can reduce the github file size, clone and push speed.







------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"boss"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月25日(星期三) 凌晨0:10
收件人:&nbsp;"hzcx998/XBook"<XBook@noreply.github.com&gt;;
抄送:&nbsp;"胡自成"<2323168280@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [hzcx998/XBook] Problem running the version download from the website  (#10)





 
Yes but remember I'm using the downloaded version, if I make the changes to the source code and do 'make' again the c.img will not be created as it's creation is commented on the makefile
